impl_core/src/main/java/io/opencensus/implcore/trace/propagation/PropagationComponentImpl.java

```diff
@@ -18,13 +18,20 @@
 
 import io.opencensus.trace.propagation.BinaryFormat;
 import io.opencensus.trace.propagation.PropagationComponent;
+import io.opencensus.trace.propagation.TextFormat;
 
 /** Implementation of the {@link PropagationComponent}. */
 public class PropagationComponentImpl extends PropagationComponent {
   private final BinaryFormat binaryFormat = new BinaryFormatImpl();
+  private final B3Format b3Format = new B3Format();
 
   @Override
   public BinaryFormat getBinaryFormat() {
     return binaryFormat;
   }
+
+  @Override
+  public TextFormat getB3Format() {
+    return b3Format;
```

Comment:
Add a test that you return the right instance class.

Reply:
Done.